core/src/main/scala/org/bitcoins/core/script/ScriptProgram.scala

```diff
@@ -61,7 +61,11 @@ case class PreExecutionScriptProgram(
     originalScript: List[ScriptToken],
     altStack: List[ScriptToken],
     flags: Seq[ScriptFlag])
-    extends ScriptProgram
+    extends ScriptProgram {
+  def addFlags(newFlags: Seq[ScriptFlag]): PreExecutionScriptProgram = {
+    this.copy(flags = newFlags)
```

Comment:
This isn't really adding is it? It's replacing the existing flags. Why not just use `.copy()` directly?